_posts/2013-04-02-practices-windows.md

```diff
@@ -39,26 +52,31 @@ Then, to start the automatic configuration, run:
 ember windows
 {% endhighlight %}
 
-##### Node Cli Tool
+[Readmore about this from the Microsoft DX Open Source team](http://felixrieseberg.com/improved-ember-cli-performance-with-windows/)
 
-The automatic configuration tool can also be installed directly, making it
-available in PowerShell & CMD. To install, run:
+#### Enabling symlinks
 
-{% highlight bash %}
-npm install ember-cli-windows -g
-{% endhighlight %}
+In order to create symlinks the account running Ember CLI must have the
+`SeCreateSymbolicLinkPrivilege`. Users in the Administrators group have this
+permission already. However if UAC (User Access Control) is enabled users in
+the Administrators group must run their shell using `Run As Administrator`.
+This is because UAC strips away certain permissions from the Administrators
+group, including `SeCreateSymbolicLinkPrivilege`.
 
-Once the tool is installed, you can run it in any Ember Cli project directory.
+![Run As Administrator]({{ site.url }}/assets/images/common-issues/run-as-admin.png)
 
-{% highlight bash %}
-ember-cli-windows
-{% endhighlight %}
+If the user account is not part of the Administrators group you will need to
+add the `SeCreateSymbolicLinkPrivilege` in order to allow the creation of
+symlinks. To do this open the `Local Security Policy` by typing Local Security
```

Comment:
If the user account is not part of the Administrators group you will need to
add the `SeCreateSymbolicLinkPrivilege` to allow the creation of symlinks.